app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -65,6 +65,8 @@ def rrd_status
   # Fetch all claims from EVSS
   # @return [Boolean] whether there are any open EP 020's
   def pending_eps?
+    return true if form.dig('rrd_metadata', 'offramp_reason')&.upcase == 'PENDING_EP'
+
```

Comment:
Since `pending_ep` is being set on line 71, you know the exact string:
```suggestion
    return true if form.dig('rrd_metadata', 'offramp_reason') == 'pending_ep'

```
which should now probably be a constant.

Reply:
Reminder to use a constant, which will avoid the need to `upcase`.